plone/app/contenttypes/browser/templates/image_view_fullscreen.pt

```diff
@@ -32,6 +38,7 @@
       border: 0;
       display:block;
       margin:0 auto;
+      max-width: 100%; height: auto;
```

Comment:
We could, but these two go together like strawberries & cream... one is not very good without the other